README.md

```diff
@@ -28,14 +28,46 @@ Offering this in bpfd allows for XDP applications whose loader was not using lib
 We are also hoping to find a way for applications linked with libxdp to use bpfd instead if it's
 in use in the system.
 
-## Development Requirements
+## Development Environment Setup
+
+- [Rust Stable & Rust Nightly](https://www.rust-lang.org/tools/install) 
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+export PATH="$HOME/.cargo/bin:$PATH"
+rustup default nightly
+rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
+```
+- LLVM 11 or later (Linux package managers should provide a recent enough release)
+
+```shell
+sudo dnf install llvm-devel clang-devel
+```
 
-- Rust Stable & Rust Nightly
 - [bpf-linker](https://github.com/aya-rs/bpf-linker)
-- protoc
-- LLVM 11 or later
+
+```shell
+cargo install bpf-linker
+```
+
+- [protoc](https://grpc.io/docs/protoc-installation/)
+
+```shell
+sudo dnf install protobuf-compiler
+```
+
+- libelf devel
+
+```shell
+sudo dnf install elfutils-libelf-devel
+```
+
 - ... and make sure the submodules are checked out
 
+```shell
+cargo update
```

Comment:
This will actually bump the deps which you don't want to do.

The statement a iut submodules can be removed anyway since they are gone now.

Reply:
I saw that submodules in the Cargo but wasnt sure if it applied to deps. 

I ran into a build issue without bumbping the `dave-tucker/aya?branch=bpfd#dc4b51ff` branch.

```
brent@cluster-a:~/bpfd$ cargo build
    Updating git repository `https://github.com/dave-tucker/aya`
error: failed to load source for dependency `aya`

Caused by:
  Unable to update https://github.com/dave-tucker/aya?branch=bpfd#dc4b51ff

Caused by:
  object not found - no match for id (dc4b51ff3ab405b5afdbe812b3a20785eea7cd84); class=Odb (9); code=NotFound (-3)
brent@cluster-a:~/bpfd$ cargo update
    Updating git repository `https://github.com/dave-tucker/aya`
    Updating crates.io index
    Updating aya v0.11.0 (https://github.com/dave-tucker/aya?branch=bpfd#dc4b51ff) -> #004f3dd6
    Updating base64ct v1.5.0 -> v1.5.1
    Updating smallvec v1.8.0 -> v1.8.1
    Updating tracing-core v0.1.27 -> v0.1.28
brent@cluster-a:~/bpfd$ cargo build
```

Reply:
I ran `cargo update` to fix (even though he said not too).

Reply:
Will hopefully be fixed now in `main` :laughing:

Reply:
The patch fixed it!